src/components/PeopleFilter/PeopleFilters.tsx

```diff
@@ -0,0 +1,166 @@
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { SexFilter } from '../../types/SexFilter';
+import { CenturiesFilter } from '../../types/CenturiesFilter';
+import { AVAILABLE_CENTURIES } from '../../constants/availableCenturies';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const peopleSex = searchParams.get('sex') || SexFilter.All;
+  const centuries = searchParams.getAll('centuries') || [];
+  const query = searchParams.get('query') || '';
+
+  const handleFilterSex = (sex: SexFilter) => {
+    const params = new URLSearchParams(searchParams);
+
+    if (sex) {
+      params.set('sex', sex);
+    } else {
+      params.delete('sex');
+    }
+
+    return params.toString();
+  };
+
+  const handleFilterBornInXCentury = (century: CenturiesFilter) => {
```

Comment:
Could you double-check if this function name is the best choice? It seems a little unclear to me.

Reply:
Is `handleFilterByBorningInSomeCentury` or `handleFilterByBorningInSelectedCenturies` better?

Of course we can also crease the most clear description like `handleFiltrationOfPeopleListBySelectedCenturiesWhichIncludesABirthYearOfSuitablePeople`, is it clear enough?

Reply:
You're trying to be funny, huh? That's a good one. Pick whatever option you like. I'm going with the last one.